ci: make LuaJIT integration workflow rerunnable

igormunkin opened the pull request:
If Tarantool tests fail in LuaJIT integration workflow used in
tarantool/luajit repository and LuaJIT submodule is already bumped in
tarantool/tarantool repository, one can't rerun the failed job since
"LuaJIT bump" step brings down the whole pipeline with "nothing to
commit" reason.

There are still many flaky tests, so to make these integration workflow
rerunnable from tarantool/luajit repository --allow-empty option is
added to git commit command.

NO_DOC=ci
NO_TEST=ci
NO_CHANGELOG=ci

ylobankov reviewed (approved):
LGTM

Buristan reviewed (approved):
Hi, Igor!
Thanks for the patch!

LGTM, except a few typos below regarding to the commit message.

> If Tarantool tests fail in LuaJIT integration workflow used in
tarantool/luajit repository and LuaJIT submodule is already bumped in
tarantool/tarantool repository, one can't rerun the failed job since
"LuaJIT bump" step brings down the whole pipeline with "nothing to
commit" reason.
>
> There are still many flaky tests, so to make these integration workflow

s/workflow/workflows/
> rerunnable from tarantool/luajit repository --allow-empty option is
added to git commit command.

s/git commit command/the git commit command/

igormunkin commented:
Sergey,

Thanks for your review!

> Hi, Igor! Thanks for the patch!
> 
> LGTM, except a few typos below regarding to the commit message.
> 
> > If Tarantool tests fail in LuaJIT integration workflow used in
> > tarantool/luajit repository and LuaJIT submodule is already bumped in
> > tarantool/tarantool repository, one can't rerun the failed job since
> > "LuaJIT bump" step brings down the whole pipeline with "nothing to
> > commit" reason.
> > There are still many flaky tests, so to make these integration workflow
> 
> s/workflow/workflows/

Fixed s/these/this/ instead.

> 
> > rerunnable from tarantool/luajit repository --allow-empty option is
> > added to git commit command.
> 
> s/git commit command/the git commit command/

Fixed.

igormunkin commented:
Yaroslav,

Thanks for your review!

> LGTM

Added your tag: `Reviewed-by: Yaroslav Lobankov <y.lobankov@tarantool.org>`

igormunkin commented:
Also backported to 2.10 in scope of 35d5bdb (2.10.0-150-g35d5bdb88) and to 1.10 in scope of 8ae0586 (1.10.13-74-g8ae0586f4).
